rasterio/_example.pyx

```diff
@@ -1,10 +1,10 @@
 # cython: boundscheck=False
 
-import numpy
-cimport numpy
+import numpy as np
+cimport numpy as np
```

Comment:
@perrygeo are these silently required for the `char[:, :, :]` parameters, or just as an example of the `import` / `cimport` pattern for pulling in numpy?  If the latter, might be worth a tiny comment to indicate such to avoid the confusion mentioned by @eseglem .